internal/xerrors/join.go

```diff
@@ -43,3 +43,8 @@ func (errs joinError) Is(target error) bool {
 	}
 	return false
 }
+
+func (errs joinError) Unwrap() []error {
```

Comment:
can you add a small test for this feature?

Reply:
You can add file join_go1.20_test.go with build tags 
```
//go:build go1.20
// +build go1.20
```
for testing true behaviour with standard package `errors`

Reply:
test added.